RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -8,6 +8,16 @@ output:
     toc_float: true
 ---
 
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Create a `DESeq2` data set from a `SummarizedExperiment`
```

Comment:
```suggestion
- Create a `DESeqDataSet` from a `SummarizedExperiment`
```

Reply:
I think I'd rather leave this one as is for the objective; I was trying to stick with words where I thought it made sense.